apis/swagger.yml

```diff
@@ -1186,9 +1186,7 @@ definitions:
         additionalProperties:
           type: "string"
       State:
-        type: "string"
```

Comment:
We need to be very careful to edit swagger.yml which defines API. I agree with your ideas totally.
From my point of view, this pr added feature in cli side. The first question we have is that can we use the current API to make it. If not, we can make API daemon side provide more data in the current API. If still not, we can change API at that time.

In this PR, change on type of `State` is not what we want if we can find other solutions.